pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
+        Parameters
+        ----------
+        rule : str
+            The offset string or object representing target conversion.
```

Comment:
I find this a bit cryptic. I think we can find a better explanation. And I'm sure there is somewhere in the pandas documentation where it explains which are the different values, we could link it. Also we could provide an example.